### PR TITLE
Make sure scale.min doesn't exceed scale.max

### PR DESCRIPTION
**Pull request by antonmoiseev**
In order to address issue https://github.com/zeit/now-cli/issues/1423

### CODE REVIEW

File: deployment/config.js [hunk 1]
```diff
@@ -45,9 +45,9 @@ module.exports = {
 			type: 'object',
 			patternProperties: {
 				'.+': {
-					type: 'object',
-					required: ['max', 'min'],
-					properties: {
+					'type': 'object',
```
Comment (antonmoiseev) [inline deployment/config.js]: Weirdly but eslint complains with quote-props error unless I wrap three following keys into quotes:
image

Comment (TooTallNate) [reply deployment/config.js]: It's because once you quote one property, it wants you to quote _all_ the properties in that object "for consistency"

Comment (antonmoiseev) [reply deployment/config.js]: I see. But since `if` is a reserved word, eslint requires it to be quoted.

**Comment by TooTallNate** [comment]
Thank you @antonmoiseev!

**Pull request merged**